catalog-info.yaml

```diff
@@ -18,19 +22,97 @@ metadata:
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: logstash-filter-elastic-integration-pull-request-pipeline
+  description: 'A pipeline for Logstash filter for Elastic integration Pull Requests'
+  links:
+    - title: 'Logstash filter for Elastic integration Pull Request pipeline'
+      url: https://buildkite.com/elastic/logstash-filter-elastic-integration-pull-request-pipeline
+spec:
+  owner: group:ingest-fp
+  type: buildkite-pipeline
+  system: buildkite
   implementation:
     apiVersion: buildkite.elastic.dev/v1
     kind: Pipeline
     metadata:
-      description: ':logstash: The logstash-filter-elastic_integration :pipeline:'
-      name: logstash-filter-elastic_integration-ci
+      description: ':logstash: The logstash-filter-elastic_integration-pull-request :pipeline:'
+      name: logstash-filter-elastic_integration-pull-request
     spec:
-      pipeline_file: .buildkite/pipeline.yml
-      provider_settings:
-        build_pull_request_forks: true
+      pipeline_file: .buildkite/pull-request-pipeline.yml
       repository: elastic/logstash-filter-elastic_integration
+      provider_settings:
+        build_pull_requests: true
+        build_pull_request_forks: false
+        build_branches: false
+        build_tags: false
+        filter_enabled: true
+        filter_condition: >-
+          build.creator.name == 'elasticmachine' && build.pull_request.id != null
+      env:
+        ELASTIC_PR_COMMENTS_ENABLED: 'true'
+        ELASTIC_SLACK_NOTIFICATIONS_ENABLED: 'true'
+        SLACK_NOTIFICATIONS_CHANNEL: '#logstash-build'
+        SLACK_NOTIFICATIONS_ON_SUCCESS: 'false'
       teams:
         ingest-fp:
-          access_level: BUILD_AND_READ
-  owner: group:ingestion-team
+          access_level: MANAGE_BUILD_AND_READ
+        everyone:
+          access_level: READ_ONLY
+
+
+# ***********************************
+# SECTION START: Plugin Build against Elasticsearch versions pipeline
+# ***********************************
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: logstash-filter-elastic-integration-build-pipeline
+  description: 'A pipeline for Logstash filter for Elastic integration build against Elasticsearch versions'
+  links:
+    - title: 'Logstash filter for Elastic integration build against Elasticsearch versions pipeline'
+      url: https://buildkite.com/elastic/logstash-filter-elastic-integration-build-pipeline
+spec:
+  owner: group:ingest-fp
   type: buildkite-pipeline
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      description: ':logstash: The logstash-filter-elastic_integration-build :pipeline:'
+      name: logstash-filter-elastic_integration-build
+    spec:
+      pipeline_file: .buildkite/build-pipeline.yml
```

Comment:
same comment here like the previous pipeline definition, let's add a `maximum_timeout_in_minutes` as well here.